projects/components/src/toggle-group/toggle-group.component.test.ts

```diff
@@ -0,0 +1,57 @@
+import { fakeAsync } from '@angular/core/testing';
+import { NavigationService } from '@hypertrace/common';
+import { ToggleGroupComponent, ToggleGroupModule, ToggleItem } from '@hypertrace/components';
+import { byText, createHostFactory, mockProvider, Spectator } from '@ngneat/spectator/jest';
+import { EMPTY } from 'rxjs';
+
+describe('Toggle Group Component', () => {
+  let spectator: Spectator<ToggleGroupComponent>;
+
+  const createHost = createHostFactory({
+    component: ToggleGroupComponent,
+    declareComponent: false,
```

Comment:
Please convert this to a shallow test and remove the import - that should also remove the need to mock nav service, which I don't see being used by this test.